src/renderer/components/cluster-manager/cluster-manager.scss

```
@@ -27,7 +27,7 @@
   }
 
   .error {
-    z-index: 1;
+    z-index: var(--z-index-base);
```

Seems like it's used with `position: relative` and this name probably more fits:

```suggestion
    z-index: var(--z-index-above);
```